puffin/src/protocol.rs

```diff
@@ -58,11 +57,6 @@ pub trait ProtocolBehavior: 'static {
     /// Get the signature which is used in the protocol
     fn signature() -> &'static Signature;
 
-    /// Gets the registry for concrete programs-under-test.
```

Comment:
Was this function unused?

Reply:
Yes, it was. We mostly referred directly to TLS_PUT_REGISTRY and SSH_PUT_REGISTRY.

I chose to not bind the protocol to a specific registry, because we will be registering the C PUTs dynamically and I would like to stay flexible on which registry is used where.